library/Xi/Sms/SmsMessage.php

```diff
@@ -34,22 +34,13 @@ class SmsMessage
      * @param string $from
      * @param array|string $to
      */
-    public function __construct($body = null, $from = null, $to = array())
+    public function __construct($body, $from, $to)
     {
         $this->body = $body;
         $this->from = $from;
 
-        if ($to) {
-            $this->setTo($to);
-        }
-    }
-
-    /**
-     * @param string $body
-     */
-    public function setBody($body)
-    {
-        $this->body = $body;
+        $to_array = $this->checkTo($to);
```

Comment:
Could $to_array and checkTo() be replaced with just an array cast?

    $this->to = (array)$to;